### DISCUSSION TITLE
Configure keyboard displacement in drag gesture

### DISCUSSION THREAD

**Discussion by markjaniczak** [pmndrs/use-gesture#575] [Ideas] [answered]
I would like to configure the base displacement when using the arrow keys to emulate the drag gesture.

My use case is that users can drag a marker along a one dimensional timeline that represents a video feed. They are able to drag the marker to a position on the timeline but afterwards they may want finely displace the marker by only one video frame at a time which they would prefer to do with arrow keys. Depending on their 'zoom' level and the duration of the video; one video frame can be represented by an arbitrary amount of pixels whereas currently keyboard controls displace by a fixed amount of 10px.

I could do this in the gesture handler by checking if the original event type is keydown and using my own displacement values rather than the gesture's but it does seem like something that should be configurable.

**Comment by dbismut** [comment, marked as answer]
Makes sense and easy to do. Post a bug and I'll mark it as enhancement. I'll do this when I have time (but feel free to propose a PR).
